documentation/modules/oauth/proc-oauth-kafka-config.adoc

```diff
@@ -109,6 +111,8 @@ An authorization server may require a client to specify the audience.
 In this case it is `kafka`.
 <6> (Optional) The connect timeout in seconds when connecting to the authorization server. The default value is 60.
 <7> (Optional) The read timeout in seconds when connecting to the authorization server. The default value is 60.
+<8> (Optional) The maximum number of retries to perform if the request to the authorization server fails in unexpected way. The default value is 0 (no retries). When using this option consider reducing the timeout times for `connectTimeoutSeconds` and `readTimeoutSeconds`. Note that retries prevent the current worker thread from being available to other requests. Many of them stalling could make the Kafka Bridge unresponsive.
```

Comment:
```suggestion
<8> (Optional) The maximum number of times to retry a failed HTTP request to the authorization server. The default value is `0`, meaning that no retries are performed. To use this option effectively, consider reducing the timeout times for the `connectTimeoutSeconds` and `readTimeoutSeconds` options. However, note that retries may prevent the current worker thread from being available to other requests, and if too many requests stall, it could make the Kafka broker unresponsive.
```